Modules/Core/Common/include/itkImageBase.h

```diff
@@ -398,15 +398,15 @@ class ITK_TEMPLATE_EXPORT ImageBase:public DataObject
   virtual void SetSpacing(const double spacing[VImageDimension]);
   virtual void SetSpacing(const float spacing[VImageDimension]);
 
-  /** Get the index (discrete) of a voxel from a physical point.
+  /** Returns the index (discrete) of a voxel from a physical point.
    * Floating point index results are rounded to integers
-   * Returns true if the resulting index is within the image, false otherwise
```

Comment:
Minor changes request to documentation.  I think it is worth explicitly noting that these functions do not verify that the resulting indices fall within the image.

Reply:
@hjmjohnson Please check the added note:

https://github.com/InsightSoftwareConsortium/ITK/blob/ec7e47678843eddf7ea45d9b83bedaadd24d1c99/Modules/Core/Common/include/itkImageBase.h#L403

https://github.com/InsightSoftwareConsortium/ITK/blob/ec7e47678843eddf7ea45d9b83bedaadd24d1c99/Modules/Core/Common/include/itkImageBase.h#L452 (copied from the previous note, but adapted code snippets)